pkg/styra/client.go

```diff
@@ -62,6 +62,9 @@ type ClientInterface interface {
 
 	DeleteDatasource(ctx context.Context, id string) (*DeleteDatasourceResponse, error)
 
+	GetLibrary(ctx context.Context, id string) (*GetLibraryResponse, error)
+	UpsertLibrary(ctx context.Context, id string, reqBody *UpsertLibraryRequest) (*UpsertLibraryResponse, error)
```

Comment:
```suggestion
	UpsertLibrary(ctx context.Context, id string, request *UpsertLibraryRequest) (*UpsertLibraryResponse, error)
```

We usually call it "request" and not "reqBody"